### ISSUE TITLE
Problem with 2.1 Pairwise sequence alignment

### ISSUE THREAD

**Issue by ETaSky** [gregcaporaso/An-Introduction-To-Applied-Bioinformatics#161]
Dear Dr. Caporaso,

I am able to pip install https://github.com/gregcaporaso/An-Introduction-To-Applied-Bioinformatics/archive/0.1.1.tar.gz. But now I have another issue when working with 2.1 Pairwise sequence alignment.
An error occurred in 2.1.2 A simple procedure for aligning a pair of sequences when I tried to run "from iab.algorithms import format_matrix"
The error message is 
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-f830f4e48990> in <module>()
----> 1 from iab.algorithms import format_matrix

ImportError: No module named iab.algorithms

So it seems to fail to find iab.algorithms.
Thank you very much!

Jincheng

**Comment by ETaSky** [comment]
I managed to solve this problem. It turned out that the pip install command did not successfully install the IAB package. I am using an Windows 7 machine and using Anaconda to implement python.
What I did that make it work is download the package from  https://github.com/gregcaporaso/An-Introduction-To-Applied-Bioinformatics/archive/0.1.1.tar.gz , and unzipped the package and within Anaconda in the unzipped folder type: 
python setup.py install
After install the package, the ipython notebook need to be restarted.
Thanks a lot!

Jincheng
